test/index.js

```diff
@@ -43,6 +43,27 @@ describe('librato', function () {
     });
   });
 
+  it('should send a prefixed message', function () {
+    const post = sinon.stub().resolves();
+    const prefix = 'prefix';
+    const librato = proxyquire('..', { 'request-promise-native': { post } })({ email, token, prefix });
+    const name = 'test.metric';
+    const tags = { tag: 'something' };
+    const value = 42;
+    librato.queue(name, tags, value);
```

Comment:
maybe some of the stubbing above could go in a `beforeEach()` or just in the outer scope under the `describe`...  There's a lot of duplicated code?

Reply:
Might even need another level of indentation with a new `describe('happy cases', function() ...`.  Anyway see what you can come up with?